### PR TITLE
Introduce Liveness API resource for MI

### PR DESCRIPTION
**Pull request by rosensilva**
New API Resource: http://localhost:9201/liveness

## User stories

### CODE REVIEW

**Review by GDLMadushanka — APPROVED** [review]

File: components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.probes/src/main/java/org/wso2/micro/integrator/probes/LivenessProbe.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (arunans23) [inline components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.probes/src/main/java/org/wso2/micro/integrator/probes/LivenessProbe.java:2]: 2023

Comment (rosensilva) [reply components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.probes/src/main/java/org/wso2/micro/integrator/probes/LivenessProbe.java]: fixed

**Comment by arunans23** [comment]
Shall we add an entry in the [wrapper.conf ](https://github.com/wso2/micro-integrator/blob/master/distribution/src/conf/wrapper.conf) as well?

**Comment by rosensilva** [comment]
@arunans23 added to the wrapper.conf

**Review by arunans23 — APPROVED** [review]

**Pull request merged**